echoregions/lines/lines.py

```diff
@@ -267,11 +280,17 @@ def filter_bottom(bottom, start_date, end_date):
             # bottom: False, otherwise: True
             bottom_mask = depth_da < bottom_da
 
+            # Reset bottom_contours index so that time index becomes time column
+            bottom_contours = bottom_contours.reset_index()
+
         else:
-            # set everything to False
+            # Set everything to False
             bottom_mask = xr.full_like(da_Sv, False)
 
-        # bottom: False becomes 0, otherwise: True becomes 1
+            # Set bottom contours to empty DataFrame with time and depth columns
+            bottom_contours = pd.DataFrame(columns=["depth", "time"])
+
+        # Bottom: False becomes 0, otherwise: True becomes 1
```

Comment:
Changing the wording to disambiguate. 
```suggestion
        # Bottom: True becomes 1, False becomes 0
```